R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -776,6 +778,19 @@ newDF <- createDataFrame(data.frame(x = c(1.5, 3.2)))
 head(predict(isoregModel, newDF))
 ```
 
+#### Decision Tree
+
+`spark.decisionTree` fits a [decision tree](https://en.wikipedia.org/wiki/Decision_tree_learning) classification or regression model on a `SparkDataFrame`.
+Users can call `summary` to get a summary of the fitted model, `predict` to make predictions, and `write.ml`/`read.ml` to save/load fitted models.
+
+We use the `longley` dataset to train a decision tree and make predictions:
+
+```{r, warning=FALSE}
+df <- createDataFrame(longley)
```

Comment:
actually, I think there's a confusion - I don't mean to change not to use `.` in formula
I mean the reason why we have `warning=FALSE` here is because `createDataFrame(longley)` will cause a warning because it has column with name with `.` in it. And we should avoid that if we can